bodhi/server/scripts/check_policies.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# Copyright ® 2017 Red Hat, Inc.
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""
+Check the enforced policies by Greenwave for each open update.
+
+Ideally, this should be done in a fedmsg consumer but we currently do not have any
+messages in the message bus yet.
+"""
+import click
+
+from sqlalchemy.sql.expression import false
+from bodhi.server import config, initialize_db, models, Session
+from bodhi.server.util import greenwave_api_post
+
+
+@click.command()
+@click.version_option(message='%(version)s')
+def check():
+    """Check the enforced policies by Greenwave for each open update."""
+    initialize_db(config.config)
+    session = Session()
+    updates = models.Update.query.filter(models.Update.pushed == false())\
+        .filter(models.Update.status.in_(
+                [models.UpdateStatus.pending, models.UpdateStatus.testing]))
```

Comment:
Can it also say "no" after it says "yes"? If so, then I think this code is fine. If not, I think the ```ci_status``` field (potentially renamed if you want) is still useful, because we can just query for updates that are not in "passed" or "skipped" (i.e., just leave off "failed").